### PR TITLE
scylla: integration tests stab 1

### PR DESCRIPTION
**Pull request by dahankzter**
Fixes: #37

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  node_2:
+    image: ${SCYLLA_IMAGE}
+    privileged: true
+    command: --smp 2 --memory 512M --seeds 192.168.100.11
+    networks:
+      public:
+        ipv4_address: 192.168.100.12
+
+#  node_3:
```
[the remark this answers is not in the document]
Comment (dahankzter) [reply docker-compose.yml:20]: I couldn't run 3 nodes even locally with the needed memory settings.

File: .travis.yml [hunk 2]
```diff
@@ -3,11 +3,20 @@ language: go
...
+  - docker
+
 cache:
   directories:
     - $HOME/.ccm/repository
     - $HOME/.local/lib/python2.7
 
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
```
[the remark this answers is not in the document]
Comment (dahankzter) [reply .travis.yml:16]: This is what Travis recommends but I guess we could change it.

File: integration.sh [hunk 3]
```diff
@@ -92,4 +92,79 @@ function run_tests() {
 	ccm remove
 }
 
-run_tests $1 $2
+function join_by {
```
[the remark this answers is not in the document]
Comment (dahankzter) [reply integration.sh:95]: How do you mean?

File: .travis.yml [hunk 4]
```diff
@@ -18,6 +27,7 @@ branches:
 env:
   global:
     - GOMAXPROCS=2
+    - DOCKER_COMPOSE_VERSION=1.25.5
   matrix:
```
[the remark this answers is not in the document]
Comment (dahankzter) [reply .travis.yml:31]: Really? Do we really want to risk backwards comp?

File: .travis.yml [hunk 5]
```diff
@@ -29,16 +39,17 @@ env:
       AUTH=false
     - CASS=3.11.4
       AUTH=false
+    - CASS=scylladb/scylla:4.0.rc1
+      AUTH=false
 
 go:
-  - 1.12.x
-  - 1.13.x
+  - 1.14.x
```
[the remark this answers is not in the document]
Comment (dahankzter) [reply .travis.yml:46]: Is there such meta tag?

File: docker-compose.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mmatczuk) [reply docker-compose.yml:20]: Maybe delete it.

File: integration.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (mmatczuk) [reply integration.sh:95]: Rewrite all that bash to an Integration tests suite runner in go.

**Pull request merged**